docs/core/tutorials/netcore-hosting.md

```diff
@@ -0,0 +1,143 @@
+
+To make hosting on Unix platforms easier, a set of more platform-neutral hosting API wrappers are available in [coreclrhost.h](https://github.com/dotnet/coreclr/blob/master/src/coreclr/hosts/inc/coreclrhost.h).
+
+An example of using coreclrhost.h (instead of mscoree.h directly) can be seen in the [UnixCoreRun host](https://github.com/dotnet/coreclr/tree/master/src/coreclr/hosts). The steps to use the APIs from coreclrhost.h to host the runtime are similar to the steps when using mscoree.h:
+
+1. Identify the managed code to execute (from command line parameters, for example). 
+2. Load the CoreCLR library.
+	1. `dlopen("./libcoreclr.so", RTLD_NOW | RTLD_LOCAL);` 
+3. Get function pointers to CoreCLR's `coreclr_initialize`, `coreclr_create_delegate`, `coreclr_execute_assembly`, and `coreclr_shutdown` functions using `dlsym`
+	1. `coreclr_initialize_ptr coreclr_initialize = (coreclr_initialize_ptr)dlsym(coreclrLib, "coreclr_initialize");`
+4. Set up AppDomain properties (such as the TPA list). This is the same as step 5 from the mscoree workflow, above.
+5. Use `coreclr_initialize` to start the runtime and create an AppDomain. This will also create a `hostHandle` pointer that will be used in future hosting calls.
+	1. Note that this function performs the roles of both steps 4 and 6 from the previous workflow. 
+6. Use either `coreclr_execute_assembly` or `coreclr_create_delegate` to execute managed code. These functions are analogous to mscoree's `ExecuteAssembly` and `CreateDelegate` functions from step 7 of the previous workflow.
+7. Use `coreclr_shutdown` to unload the AppDomain and shut down the runtime. 
+
+## Conclusion
+Once your host is built, it can be tested by running it from the command line and passing any arguments (like the managed app to run) the host expects. When specifying the .NET Core app for the host to run, be sure to use the .dll that is produced by `dotnet build`. Executables produced by `dotnet publish` for self-contained applications are actually the default .NET Core host (so that the app can be launched directly from the command line in mainline scenarios); user code is compiled into a dll of the same name. 
+
+If things don't work initially, double-check that *coreclr.dll* is available in the location expected by the host, that all necessary Framework libraries are in the TPA list, and that CoreCLR's bitness (32- or 64-bit) matches how the host was built.
+
+Hosting the .NET Core runtime is an advanced scenario that many developers won't require, but for those who need to launch managed code from a native process, or who need more control over the .NET Core runtime's behavior, it can be very useful. Because .NET Core is able to run side-by-side with itself, it's even possible to create hosts which initialize and start multiple versions of the .NET Core runtime and execute apps on all of them in the same process. 
```

Comment:
Good idea, Maira. It overlaps some with the second paragraph in the introduction, so instead of moving this whole paragraph I just updated that one with a new opening that explicitly calls hosting an advanced scenario.